packages/twenty-server/src/core/workspace/workspace.entity.ts

```diff
@@ -54,6 +55,11 @@ export class Workspace {
   @OneToMany(() => User, (user) => user.defaultWorkspace)
   users: User[];
 
+  @OneToMany(() => UserWorkspace, (userWorkspace) => userWorkspace.workspace, {
+    onDelete: 'CASCADE',
+  })
+  workspaceUsers: UserWorkspace[];
```

Comment:
```suggestion
  users: UserWorkspace[];
```

Reply:
Can't change it to `users`, one already exists. 

https://github.com/twentyhq/twenty/blob/b69fed90232ce54fd2e05ccb6456d35f9a3841f8/packages/twenty-server/src/core/workspace/workspace.entity.ts#L54-L55

Do you want me to name it to something else? or should I keep `workspaceUsers`? Let me know :)